src/main/kotlin/step4/view/ResultView.kt

```diff
@@ -0,0 +1,36 @@
+package step4.view
+
+import step4.model.Car
+
+object ResultView {
+    private const val CAR_NAME_SEPARATOR = " : "
+    private const val ONE_BLOCK = "-"
+
+    fun output(numberOfTrials: Int, cars: List<Car>) {
+        for (i in 0 until numberOfTrials) {
+            val actualTrial = i + 1
+
+            println("trial$CAR_NAME_SEPARATOR$actualTrial")
+            outputForTrials(actualTrial, cars)
+        }
+    }
+
+    fun noticeWinner(winner: String) {
+        println("${winner}가 최종 우승했습니다.")
+    }
+
+    private fun outputForTrials(actualTrial: Int, cars: List<Car>) {
+        for (element in cars) {
+            println("${element.name}$CAR_NAME_SEPARATOR" + visualize(actualTrial, element.raceHistory))
+        }
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        cars.forEach {
            println("${it.name}$CAR_NAME_SEPARATOR${visualize(actualTrial, it.raceHistory)}")
        }
```

Reply:
반영하였습니다!